File: examples/legacy-architecture-samples/simple-stack-example-mvvm-fragments/src/main/java/com/zhuinden/simplestackexamplemvvm/application/injection/Injector.java
package com.zhuinden.simplestackexamplemvvm.application.injection;

import com.zhuinden.simplestackexamplemvvm.application.CustomApplication;

/**
 * Created by Zhuinden on 2017.07.25..
 */

public class Injector {
    public static ApplicationComponent get() {
        return CustomApplication.get().appComponent();
    }
}

File: samples/legacy-architecture-samples/simple-stack-example-mvvm-fragments/src/main/java/com/zhuinden/simplestackexamplemvvm/application/injection/Injector.kt
package com.zhuinden.simplestackexamplemvvm.application.injection


import com.zhuinden.simplestackexamplemvvm.application.CustomApplication

/**
 * Created by Zhuinden on 2017.07.25..
 */
object Injector {
    fun get(): ApplicationComponent {
        return CustomApplication.INSTANCE!!.appComponent()
    }
}